src/building.lua

```diff
@@ -1,5 +1,3 @@
-local Util = require("util")
```

Comment:
Some of these `require`d util even though they never used it.

Reply:
They are like decorative pillows. They have no practical purpose but they make you feel comfortable.